### DISCUSSION TITLE
Certificate Error  TLS Request

### DISCUSSION THREAD

**Discussion by suqdeveloper** [community/community#85660] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Certificate Error  TLS Request

**Comment by LuksJobs** [comment]
Can you bring more information about the error?

**Comment by azupworkuser** [comment, marked as answer]
I got the answer here.
https://github.com/orgs/community/discussions/23372#discussioncomment-3240117

If above answer is wrong. Please add more details here.
